Sources/Swift/Helper/DataDeserialization.swift

```diff
@@ -1,3 +1,5 @@
+@_implementationOnly import _SentryPrivate
+
 @_spi(Private) @objc public final class DataDeserialization: NSObject {
```

Comment:
`h`: Don't we have to prefix this class with Sentry to avoid naming collisions cause it's exposed to ObjC?